src/Libraries/Microsoft.Extensions.AI.Abstractions/SpeechToText/SpeechToTextResponse.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json.Serialization;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>Represents the result of an speech to text request.</summary>
+public class SpeechToTextResponse
+{
+    /// <summary>The list of choices in the generated text response.</summary>
+    private IList<SpeechToTextMessage> _choices;
+
+    /// <summary>Initializes a new instance of the <see cref="SpeechToTextResponse"/> class.</summary>
+    /// <param name="message">the generated text representing the singular choice message in the response.</param>
+    public SpeechToTextResponse(SpeechToTextMessage message)
+        : this([Throw.IfNull(message)])
+    {
+    }
+
+    /// <summary>Initializes a new instance of the <see cref="SpeechToTextResponse"/> class.</summary>
+    /// <param name="choices">The list of choices in the response, one message per choice.</param>
+    [JsonConstructor]
+    public SpeechToTextResponse(IList<SpeechToTextMessage> choices)
```

Comment:
What does choices map to here? Does that map to the multiple inputs provided to the GetResponseAsync method? Choices is the wrong name for that, I think.

Reply:
I simplified the API, there's no more the concepts of Choices nor multiple inputs, now we have one audio giving one response with one or many contents, 99% of the time will be one `TextContent`